spec/lib/miq_expression_spec.rb

```diff
@@ -17,31 +17,62 @@
       expect(sql).to eq("\"vms\".\"name\" = 'foo'")
     end
 
+    it "generates the SQL for a = expression with expression as a value" do
+      sql, * = MiqExpression.new("=" => {"field" => "Vm-name", "value" => "Vm-name"}).to_sql
+      expect(sql).to eq("\"vms\".\"name\" = \"vms\".\"name\"")
```

Comment:
Seems valid to me. `select * from vms where vms.name=vms.name` returns all rows.